### PR TITLE
Adding some UART board definitions to match specification

### PR DESCRIPTION
**Pull request by brentru**
This pull request adds some uart board definition json files to match the specification decided on in https://github.com/adafruit/Wippersnapper_Boards/pull/115 and check the validator

### CODE REVIEW

**Comment by brentru** [comment]
@lorennorman  This is the first PR involving the UART board definitions getting added in. Could you review this PR since @tyeth  will be basing a future task from this PR?

**Review by lorennorman — CHANGES_REQUESTED** [review]

File: boards/funhouse/definition.json [hunk 1]
```diff
@@ -111,7 +111,31 @@
...
+            "displayName":"D44 (UART RX)",
+            "dataType":"bool",
+            "uartTx": true
+         },
+         {
+            "name":"D43",
+            "displayName":"D43 (UART TX)",
+            "dataType":"bool",
+            "uartTx": true
+         }
```
[the remark this answers is not in the document]
Comment (brentru) [reply boards/funhouse/definition.json]: Yes this is a typo/miss, resolved in the next 2 commits

File: boards/funhouse/definition.json [hunk 2]
```diff
@@ -111,7 +111,29 @@
...
+            "displayName":"D44 (UART RX)",
+            "dataType":"bool",
+            "uartRx": true
+         },
+         {
+            "name":"D43",
+            "displayName":"D43 (UART TX)",
+            "dataType":"bool",
+            "uartTx": true
+         }
```
Comment (lorennorman) [inline boards/funhouse/definition.json:136]: Now you've got two pins name "D43" 😅 

I need to augment our JSON validator so we can add more complex checks like these.

Comment (brentru) [reply boards/funhouse/definition.json]: Sorry, was working quickly. The first one is supposed to be D44 for its name. Put back.

**Review by lorennorman — APPROVED** [review]

**Pull request merged**